### DISCUSSION TITLE
pass flashvars to Linux desktop program

### DISCUSSION THREAD

**Discussion by ousia** [ruffle-rs/ruffle#6609] [Q&A] [answered]
Sorry for this very basic question, but how can I invoke in LInux a Flash animation including flashvars?

Flash projector allows:

```
flashplayer file://$HOME/flash.swf?variable=value
```

Many thanks for your help.

**Comment by n0samu** [comment, marked as answer]
From `ruffle --help`:
```
    -P <PARAMETERS>
            A "flashvars" parameter to provide to the movie. This can be repeated multiple times,
            for example -Pkey=value -Pfoo=bar
```

**Reply by ousia** [reply]
Many thanks for your help, @n0samu.

I thought there was no `ruffle --help`.
